file-formats/sito/sito-v1.json

```diff
@@ -0,0 +1,893 @@
+{
+  "$comment": "This file is autogenerated, do not edit manually, see https://github.com/SAP/abap-file-formats for more information.",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/sito/sito-v1.json",
+  "title": "Situation Object Properties",
+  "description": "Situation Object properties",
+  "type": "object",
+  "properties": {
+    "formatVersion": {
+      "title": "Format version",
+      "description": "Format version",
+      "type": "string",
+      "const": "1"
+    },
+    "header": {
+      "title": "Header",
+      "description": "Header",
+      "type": "object",
+      "properties": {
+        "description": {
+          "title": "Description",
+          "description": "Description of the ABAP object",
+          "type": "string",
+          "maxLength": 60
+        },
+        "originalLanguage": {
+          "title": "Original Language",
+          "description": "Original language of the ABAP object",
+          "type": "string",
+          "minLength": 2,
+          "maxLength": 2,
+          "pattern": "^[a-z]+$"
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP language version",
+          "type": "string",
+          "enum": [
+            "standard",
+            "keyUser",
+            "cloudDevelopment"
+          ],
+          "enumTitles": [
+            "Standard",
+            "ABAP for Key Users",
+            "ABAP Cloud Development"
+          ],
+          "enumDescriptions": [
+            "Standard",
+            "ABAP for key user extensibility",
+            "ABAP cloud development"
+          ],
+          "default": "standard"
+        }
+      },
+      "additionalProperties": false,
+      "required": [
+        "description",
+        "originalLanguage"
+      ]
+    },
+    "sit2Object": {
+      "title": "Situation Object",
+      "description": "Situation Object",
+      "type": "object",
+      "properties": {
+        "sitnobjectid": {
+          "title": "Situation Object ID",
+          "description": "Situation Object ID",
+          "type": "string",
+          "maxLength": 32
+        },
+        "abapLanguageVersion": {
+          "title": "ABAP Language Version",
+          "description": "ABAP Language Version",
+          "type": "string",
+          "enum": [
+            "standardAbap",
+            "abapForKeyUsers",
+            "abapForCloudDevelopment"
+          ],
+          "enumDescriptions": [
+            "Standard ABAP",
+            "ABAP for Key Users",
+            "ABAP for Cloud Development"
+          ]
+        },
+        "sitnobjisextensible": {
+          "title": "Object Extensible",
+          "description": "Extensibility Flag for Situation Objects",
+          "type": "string",
+          "enum": [
```

Comment:
In our interface, a boolean representation is CHAR(1) - as usual in ABAP.

How can we avoid that it is transformed to string instead of boolean?

Reply:
You can just use `abap_bool` as type. This will be transferred to `boolean` in the JSON schema.

Here you can see an example:
https://github.com/SAP/abap-file-formats/blob/d371351c94bbbb93dbb385d8cb8220708f57fa97/file-formats/clas/type/zif_aff_clas_v1.intf.abap#L71

We should update the documentation. I have created an issue see #365